_posts/2018-10-12-strimzi-0.8.0-release.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title:  "Strimzi 0.8.0 released!"
+date: 2018-10-12
+author: paolo_patierno
+---
+
+We are delighted to announce the new Strimzi 0.8.0 release with many awesome new features!
+
+<!--more-->
+
+# Accessing Kafka cluster from outside without TLS
+
+The previous release added the possibility to access the Kafka cluster from the outside.
+The Kafka clients were able to exchange messages when not running on the same Kubernetes/OpenShift cluster where Kafka was deployed but from the outside.
```

Comment:
_Suggested change:_

Previously, the TLS protocol needed to be enabled on the client side because all the provided `listeners` could only be accessed through encryption.